contents/articles/2017-12_object_fixation/index.md

```diff
@@ -0,0 +1,20 @@
+--- 
+title: Real-time object recognition using fixations to control prosthetic hand
+date: Fri Dec 15 2017 11:31:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Real-time object recognition with fixations to control grasps of a robotic prosthetic hand..."
+featured_img: "../../../../media/images/blog/object_detection.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/object_detection.jpg"
+---
+
+[Jesse Weisberg](https://github.com/jesseweisberg) has made real-time object detection plugin that integrated with Pupil Labs headset to indentify what object the user is currently fixated upon and control a prosthetic hand by forming grasps based on the fixated object.
```

Comment:
Replace paragraph with below:

We are excited to share a project from a Pupil community member, [Jesse Weisberg]((https://github.com/jesseweisberg). Jesse has developed a plugin for Pupil that enables a prosthetic hand to form grasps based on what the wearer is fixating on in the scene in real-time on the CPU (10-15 fps)! 

This project is a great demonstration of how to extend Pupil and contribute back to the open source community!